### ISSUE TITLE
Improve pagination visibility and reset to page 1 after leaving

### ISSUE THREAD

**Issue by SaturnusDJ** [kimai/kimai#4190]
### Describe the problem

For example, go in "All times" (https://demo.kimai.org/en_GB/team/timesheet/). Have enough records there, then go to a page using the numbers in the right bottom.
Go elsewhere in Kimai and go back to "All times": You land on the same page as before. 1. This is unusual compared to other software. 2. It is not easily visible or changeable.
This applies more strongly the longer the list is (page size in filter).

### Describe the solution you'd like

1. Do not go back to the page left before, but always land on page 1.
2. Pagination in the url and in the right top (an addition to the right bottom).

**Comment by kevinpapst** [comment]
Dozens of people wanted that feature and it makes a lot of sense: if you are cycling through pages to check stuff, you likely want to move on with that task, after checking something else.

 > This is unusual compared to other software. 

Perfect argument to stop innovations 😁 

>  It is not easily visible or changeable.

Click the X in the search box to get a clean search filter.

**Comment by SaturnusDJ** [comment]
Hmmm okay. I thought the better way to look at data is to open multiple tabs, but of course that is more difficult on mobile devices.

I think there is no X when not using the search filter but just being on let's say page 3?

**Comment by kevinpapst** [comment]
>  I think there is no X when not using the search filter but just being on let's say page 3?

True, I have to find a solution for that. Thanks for letting me know!
